### PR TITLE
OSDOCS-4667: document known issue with spare-gb feature of LVMS in microshift

### PR DESCRIPTION
**Pull request by dhellmann**
Version(s): 4.12

Issue: [OSDOCS-4667](https://issues.redhat.com//browse/OSDOCS-4667)

Link to docs preview:
https://54316--docspreview.netlify.app/microshift/latest/microshift_storage/persistent_storage_microshift/microshift-storage-plugin-overview.html

QE review:
N/A -- MicroShift is Developer Preview
SME approval is required, author is the SME approver

### CODE REVIEW

**Review by kowen-rh — APPROVED** [review]
LGTM. :+1:

File: modules/microshift-lvms-configuring.adoc [hunk 1]
```diff
@@ -36,12 +36,17 @@ device-classes: <2>
...
-<5> unit64. Storage capacity in GiB to be spared. Defaults to `10`. 
+<5> unit64. Storage capacity in GiB to be left unallocated in the volume group. Defaults to `10`. 
 <6> Boolean. Indicates that the `device-class` is used by default. Defaults to `false`. 
 <7> unit. The number of stripes in the logical volume.
 <8> String. The amount of data that is written to one device before moving to the next device. 
 <9> String. Extra arguments to pas `lvcreate`, for example, `[--type=raid1"`]. 
 +
+[WARNING]
+====
+There is a race condition that prevents LVMS from accurately tracking the allocated space and preserving the `spare-gb` for a device class when multiple PVCs are created simultaneously. Use separate volume groups and device classes to protect the storage of highly dynamic workloads from each other reliably.
```
Comment (GroceryBoyJr) [inline modules/microshift-lvms-configuring.adoc:47]: Suggestion: 
There is a race condition that prevents LVMS from accurately tracking the allocated space and preserving the `spare-gb` for a device class when multiple PVCs are simultaneously created. Use separate volume groups and device classes to reliably protect the storage of highly dynamic workloads from each other.

(move the words "simultaneously" and "reliably")

**Review by GroceryBoyJr — COMMENTED** [review]
only two small nits, please sir!

File: modules/microshift-lvms-configuring.adoc (hunk 1 above)
Comment (dhellmann) [reply modules/microshift-lvms-configuring.adoc:47]: The word "simultaneously" is significant. There is no race condition if you make a PVC, wait for the PV, then make another PVC.

Maybe "concurrently" is a better choice?

Comment (dhellmann) [reply modules/microshift-lvms-configuring.adoc]: I did remove the "reliably"

**Pull request merged**